lib/wicked_pdf/railtie.rb

```diff
@@ -6,13 +6,15 @@ class WickedPdf
   if defined?(Rails.env)
     class WickedRailtie < Rails::Railtie
       initializer 'wicked_pdf.register', :after => 'remotipart.controller_helper' do |_app|
-        if ActionController::Base.respond_to?(:prepend) &&
-           Object.method(:new).respond_to?(:super_method)
-          ActionController::Base.send :prepend, PdfHelper
-        else
-          ActionController::Base.send :include, PdfHelper
+        ActiveSupport.on_load(:action_controller) do
```

Comment:
Could this new approach break the compatibility of wicked_pdf and remotipart? I do observe a bug if wicked_pdf 2.0.x is used with remotipart. In Ruby 2.6, the bug is announced as a FrozenError. We get more information under Ruby <2.5 to see that there is indeed a loop of rendering.

Reply:
I can confirm this does break our app. I forked this repo and simply rolled back to a simpler version just to debug this and we are working again https://github.com/darrenterhune/wicked_pdf/commit/ccc861596dcd8e43e5243a79b8658cf7c3c9b4e0 that said, wicked_pdf still worked locally, it was just barfing on CircleCI with this https://github.com/mileszs/wicked_pdf/issues/891